### ISSUE TITLE
Setup continuous deployment of osm2streets-web to github.io

### ISSUE THREAD

**Issue by BudgieInWA** [a-b-street/osm2streets#26]
Like osm2lanes, we should get osm2streets-web hosted right away.

Feel free to pick this up!

**Comment by droogmic** [comment]
> It should be easy once `-web` is merged into main. I'm happy to do it!
> 
>     1. Copy https://github.com/a-b-street/osm2lanes/blob/main/.github/workflows/web.yml (probably verbatim...)
> 
>     2. Fiddle with some github repo settings to enable pages to run off the `gh-pages` branch. I hit a bit of complication when I did this in https://github.com/dabreegster/bus_spotting the other week, because github has changed some default security settings. I can dig up my workaround from there if we hit the same thing.

+ the Release.toml file

**Comment by dabreegster** [comment]
https://a-b-street.github.io/osm2streets is up!
